FBTweak/FBTweakCategory.h

```diff
@@ -50,4 +61,21 @@
  */
 - (void)removeTweakCollection:(FBTweakCollection *)tweakCollection;
 
+/**
+ @abstract Resets all tweaks in \c tweakCollections to their default value.
+ */
+- (void)reset;
+
+/**
+ @abstract Asynchronously updates \c tweakCollections to the latest value, and calls \c completion
+ when done. Upon error \c error is set, otherwise \c error is \c nil.
+ @param completion Completion block to be called when the update is complete. \c completion can be
+ called after this method returns.
+ @discussion When the update operation is complete, is it expected that the \c tweakCollection will
```

Comment:
`When the update operation is complete all \c tweakCollection's objects will be up-to-date.`?